### PR TITLE
Added existing sql procedure & added its description

### PR DESCRIPTION
**Pull request by zlochina**
Added description of my view of a given procedure. Not really came up with particular kinds of potential tests, instead expecting to resolve the questions, which were arised during inspection of the procedure

### CODE REVIEW

File: SQL/procedures/.gitkeep
Comment (F-I-D-O) [inline SQL/procedures/.gitkeep:1]: please remove this file

File: SQL/proc_descriptions/1st_procedure.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        - `Insert Into nodes(id, geom, area) Select osm_id, geom, map_area From nodes_tmp`. There is no filtering by `area`, which implicates that all data in nodes_tmp should contain only data related to given `map_area`. Could be the first thing to be tested? But testing is not about data, but procedure. Not sure what to do. (it also applicates to every other `insert`)
+        - `Insert Into ways (id, tags, geom, "from", "to", area, oneway)`. There is a more complicated `SELECT` as it is double joined on `nodes_tmp` table. Do not really see anything, that should be tested here
+        - `Insert Into nodes_ways (node_id, way_id, position, area)`. There is also a more complicated `SELECT` with double `JOIN`, but do not really see, what could be tested here. 
+- Note to the flow: inserting is executed with additional specification `On Conflict Do Nothing`. The question is: is there a case, when conflict occures, it does nothing, but actually shouldn't?
+<!-- - Additional question. As it was described above, this procedure should be executed only in a context that there are already some temporary data -->
```
Comment (F-I-D-O) [inline SQL/proc_descriptions/1st_procedure.md:17]: yes, and it won't be needed in the new pipeline most likely

File: SQL/proc_descriptions/1st_procedure.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    2) Now there is an attempt to insert new data instead, specifically data from corresponding temporary tables, which are in some way connected to `map_area`:
+        - `Insert Into nodes(id, geom, area) Select osm_id, geom, map_area From nodes_tmp`. There is no filtering by `area`, which implicates that all data in nodes_tmp should contain only data related to given `map_area`. Could be the first thing to be tested? But testing is not about data, but procedure. Not sure what to do. (it also applicates to every other `insert`)
+        - `Insert Into ways (id, tags, geom, "from", "to", area, oneway)`. There is a more complicated `SELECT` as it is double joined on `nodes_tmp` table. Do not really see anything, that should be tested here
+        - `Insert Into nodes_ways (node_id, way_id, position, area)`. There is also a more complicated `SELECT` with double `JOIN`, but do not really see, what could be tested here. 
+- Note to the flow: inserting is executed with additional specification `On Conflict Do Nothing`. The question is: is there a case, when conflict occures, it does nothing, but actually shouldn't?
```
Comment (F-I-D-O) [inline SQL/proc_descriptions/1st_procedure.md:16]: I do not recall what was the reason for the conflict handling

**Review by F-I-D-O — CHANGES_REQUESTED** [review]

File: SQL/proc_descriptions/1st_procedure.md (hunk 1 above)
Comment (zlochina) [reply SQL/proc_descriptions/1st_procedure.md:17]: Not sure about which question this answer belongs to

**Review by F-I-D-O — APPROVED** [review]

**Pull request merged**